src/arch/util/machine-smp.C

```diff
@@ -94,9 +94,7 @@ static struct CmiStateStruct Cmi_default_state; /* State structure to return dur
 #if CMK_SHARED_VARS_NT_THREADS
 
 CmiNodeLock CmiMemLock_lock;
```

Comment:
Can you add comments explaining the uses of these two very similarly named variables?

Reply:
Done